qiskit/tools/jupyter/qiskit_toolbar/main.js

```diff
@@ -0,0 +1,37 @@
+define([
+    'base/js/namespace',
+    'base/js/events'
+], function (Jupyter, events) {
+
+    // Adds a cell above current cell (will be top if no cells)
+    var add_cell = function () {
+        Jupyter.notebook.
+            insert_cell_above('code').
+            // Define default cell here
+            set_text(`import numpy as np
+# Qiskit
+from qiskit import *
+from qiskit.tools.visualization import plot_histogram
```

Comment:
(#1878)
```suggestion
from qiskit.visualization import *
```

Reply:
Fwiw, this will still work post #1878. That's a key part of that PR because the qiskit.tools.visualization api is listed as stable as of 0.7.0 we want that to keep working moving forward. We can update this at a later time to be consistent when there is less churn in the repo.